mantidproject/mantid
# Fixing location of concepts category page

Fix a bug in the concepts index page being generated.

**To test:**

```
$ cd <builddir>
$ rm -rf docs/doctrees
$ cmake . --build --target docs-html
```
Then see that you have a `docs/html/concepts/index.html` that actually lists the concepts.



*There is no associated issue*

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
